internal/stack/_static/docker-compose-stack.yml.tmpl

```diff
@@ -170,11 +170,12 @@ services:
       interval: 60s
       timeout: 50s
       retries: 5
-    command: bash -c 'if [[ ! $(bin/logstash-plugin list) == *"logstash-filter-elastic_integration"* ]]; then echo "Missing plugin logstash-filter-elastic_integration, installing now" && bin/logstash-plugin install logstash-filter-elastic_integration; fi && bin/logstash -f /usr/share/logstash/pipeline/logstash.conf'
+    # config copy is intentional that mounted volumes will be busy and cannot be overwritten
+    command: bash -c 'yes | cp /usr/share/logstash/pipeline/generated_logstash.conf /usr/share/logstash/pipeline/logstash.conf && if [[ ! $(bin/logstash-plugin list) == *"logstash-filter-elastic_integration"* ]]; then echo "Missing plugin logstash-filter-elastic_integration, installing now" && bin/logstash-plugin install logstash-filter-elastic_integration; fi && bin/logstash -f /usr/share/logstash/pipeline/logstash.conf --config.reload.automatic'
     volumes:
       - "../certs/logstash:/usr/share/logstash/config/certs"
       - "../certs/elasticsearch/cert.pem:/usr/share/logstash/config/certs/elasticsearch.pem"
-      - "./logstash.conf:/usr/share/logstash/pipeline/logstash.conf:ro"
+      - "./logstash.conf:/usr/share/logstash/pipeline/generated_logstash.conf:ro"
```

Comment:
review note: @bhapas as we synced up, I have put back pipeline config changes as it seemed to me you may need to change integrations configs and if changes are behavioral I don't think that would be a right way.
Rather focused on long term how we can overwrite config so that Logstash automatically detects. So far `pipeline/logstash.conf` was read-only and even after setting write mode, we cannot simply change it because it was mounted. With this change, I separated `elastic-package` generated config and Logstash writable/changeable config. If we change config, Logstash automatically detects (configurable every 3sec) and reloads the pipeline. This is a right approach I think especially some integrations may require special pipeline setting like yours (fingerprint on ingest doc fields).